packages/server/src/appMigrations/migrations/20240604153647_initial_sqs.ts

```diff
@@ -36,16 +36,6 @@ const migration = async () => {
 
   // at the end make sure design doc is ready
   await sdk.tables.sqs.syncDefinition()
-  // only do initial search if environment is using SQS already
```

Comment:
Why is this not needed? What about updates from a non-sqs version to this one?

Reply:
All this was doing in practice was kicking off a CouchDB replication into SQLite. It was an optimisation, it isn't a necessary step for SQS to work. I discussed this with @mike12345567 (https://discord.com/channels/733030666647765003/1022445411098710016/1273597160428146689) and we decided to get rid of it. The reason it came up is that these migrations aren't a user-triggered action, so we wouldn't be able to consult PostHog for the status of a flag here. That could lead to a mismatch in flags being disabled here but enabled elsewhere. Because of this it felt like the cleanest option was to remove this and rely on the backfill being triggered by a real search.

It's a bit academic in practice because for most users this code has already been run in Budicloud.